packages/@okta/vuepress-site/docs/reference/api/features/index.md

```diff
@@ -0,0 +1,696 @@
+    }
+  }
+}
+```
+##### Error Response
+
+An invalid `id` returns a `404 Not Found` status code.
+
+```http
+HTTP/1.1 404 Not Found
+Content-Type: application/json
+
+{
+    "errorCode": "E0000007",
+    "errorSummary": "Not found: Resource not found: invalidFeatureId (Feature)",
+    "errorLink": "E0000007",
+    "errorId": "oaenAlD87mGTuWeGi7AjmhIBA",
+    "errorCauses": []
+}
+```
+
+### List Features
+
+<ApiOperation method="get" url="/api/v1/features" />
+
+Returns a list of all available self-service Features for your org.
+
+#### Request Parameters
+
+None
+
+#### Response Body
+
+Array of [Feature](#feature-model) objects.
+
+#### Usage Example
+
+The following request returns a list of all available self-service Features:
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/features"
+```
+
+##### Response
+
+```json
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+  {
+    "id": "ftrZooGoT8b41iWRiQs7",
+    "type": "self-service",
+    "status": "DISABLED",
+    "name": "Example feature name",
+    "description": "Example feature description",
+    "stage": {
+      "value": "EA"
+    },
+    "_links": {
+      "enable": {
+        "href": "https://{yourOktaDomain}/api/v1/features/ftrZooGoT8b41iWRiQs7/enable",
+        "hints": {
+          "allow": [
+            "POST"
+          ]
+        }
+      },
+      "self": {
+        "href": "https://{yourOktaDomain}/api/v1/features/ftrZooGoT8b41iWRiQs7"
+      },
+      "dependents": {
+        "href": "https://{yourOktaDomain}/api/v1/features/ftrZooGoT8b41iWRiQs7/dependents"
+      },
+      "dependencies": {
+        "href": "https://{yourOktaDomain}/api/v1/features/ftrZooGoT8b41iWRiQs7/dependencies"
+      }
+    }
+  },
+  {
+     "id":"ftrcUG2WEt1sjXVSIok3",
+     "type":"self-service",
+     "status":"ENABLED",
+     "name":"Example feature name",
+     "description":"Example feature description",
+     "stage":{
+        "value":"BETA",
+        "state":"OPEN"
+     },
+     "_links":{
+        "helpDoc":{
+           "href":"https://https://help.okta.com/en/prod/Content/index.htm"
+        },
+        "self":{
+           "href":"https://{yourOktaDomain}/api/v1/features/ftrcUG2WEt1sjXVSIok3"
+        },
+        "survey":{
+           "href":"https://goo.gl/forms/aRelevantSurvey"
+        },
+        "dependents":{
+           "href":"https://{yourOktaDomain}/api/v1/features/ftrcUG2WEt1sjXVSIok3/dependents"
+        },
+        "dependencies":{
+           "href":"https://{yourOktaDomain}/api/v1/features/ftrcUG2WEt1sjXVSIok3/dependencies"
+        }
+     }
+  }
+]
+```
+
+### Update Feature
+
+<ApiOperation method="post" url="/api/v1/features/${featureId}/${lifecycle}" />
+
+Updates a Feature's lifecycle. Use this endpoint to enable or disable a Feature for your org.
```

Comment:
We're saying `lifecycle` here but what gets updated in the Feature object is called `status`. I understand these are generally known as lifecycle operations but notation-wise, would this cause any confusion?

Reply:
ya it should say status. refactored.